samples/README.md

```diff
@@ -0,0 +1,27 @@
+# Samples, Snippets, and How-To Guides
+
+Developers like to learn by looking at code, and so the Azure SDK comes with a myriad of code samples in the form of short code snippets, sample applications, and how-to guides. This document describes where to find all these resources.
```

Comment:
```suggestion
Developers like to learn by looking at code, and so the Azure SDK comes with many samples in the form of short code snippets, sample applications, and how-to guides. This document describes where to find all these resources.
```

Reply:
"A myriad of" is technically correct, but might get us trolled vs. just "myriad" 😂. This simplifies the language a bit.